browser/ui/brave_browser_window.h

```diff
@@ -32,6 +32,10 @@ class BraveBrowserWindow : public BrowserWindow {
       speedreader::SpeedreaderTabHelper* tab_helper,
       bool is_enabled);
 
+  // Returns the bottom position of the Shield's panel. Renderers will call this
+  // to check if the bottom of the panel exceeds the overall screen's height
+  virtual gfx::Rect GetShieldsBubbleRect() = 0;
```

Comment:
Due to this, test build seems failed. Let's provide default impl instead of pure abstract method.